src/shell.ts

```diff
@@ -733,6 +733,36 @@ export class Shell {
                 this.t.writeln("Invalid OTP, please try again")
         }
     }
+    async verifyPeer(g: Gate) {
+        if (g.verified || !g.fp)
+            return
+        this.t.writeln(`Peer ${g.name} is unverified`)
+        const { value } = await Preferences.get({ key: "alwaysVerify" })
+        if (value == "true") {
+            await this.verifyFP(g.fp)
+            return
+        }
+        const fields = [
+            { prompt: "Verify peer" },
+            { prompt: "Not now" },
+            { prompt: "Don't ask again"}
```

Comment:
A better prompt will be "Don't ask again and always verify"

Reply:
I think just "Always verify" works better.